doc/GETTING_STARTED.md

```diff
@@ -29,6 +30,14 @@ npm install libp2p
 
 If you're new to libp2p, we recommend configuring your node in stages, as this can make troubleshooting configuration issues much easier. In this guide, we'll do just that. If you're more experienced with libp2p, you may wish to jump to the [Configuration readme](./CONFIGURATION.md).
 
+## ESM
```

Comment:
```suggestion
### ESM
```

this needs to be the same level as it’s sibling headers, otherwise “Basic Setup” etc become sub-sections of “ESM”

the TOC will also need updating.